docs/CustomUIA.md

```diff
@@ -0,0 +1,125 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT Licence. -->
+
+## Custom UI Automation properties
+Axe Windows supports the inspection of [custom UI Automation properties](https://docs.microsoft.com/en-gb/windows/win32/winauto/uiauto-custompropertieseventscontrolpatterns). The custom properties for which Axe Windows should register are described in a configuration file that is loaded once per session and applies to all inspected UI Automation elements.
+
+### Configuration file format
+Custom UI Automation configuration data is specified in a [JSON](https://en.wikipedia.org/wiki/JSON#Syntax) formatted text file in the configuration directory. The configuration file consists of one object with the following possible attributes:
```

Comment:
Specifics on the config file location intentionally omitted. Exactly where should it be stored and what should it be called?

Reply:
This is an excellent question. I think my recollection from feature estimation was that we intended for Accessibility Insights for Windows to use a well-known file name/location, but for axe-windows to add "accepting that file" as part of API calls; if that's correct, I think it would be reasonable for axe-windows' doc here to omit the name/location and for that to be documented in accessibility-insights-windows.

I'm fine with deferring the specifics of filename/location/API changes for a later PR, though; it's okay for this one to just cover the format.

Reply:
The specific location will be part of changes in AIWin. The Axe.Windows CLI will eventually allow the user to specify a json file via the command line.